### PR TITLE
modified readme.md

### PR DESCRIPTION
**Pull request by wangwenyue**

### CODE REVIEW

File: drizzle/README.md [hunk 1]
```diff
@@ -49,7 +55,7 @@ truffle compile
 > Notice: We use [AppChain-Truffle-Migrate](https://github.com/cryptape/appchain-truffle-migrate) to deploy the contract, so the commend is different from truffle-box.
 
 ```shell
-npm run migrate
+yarn migrate
```
Comment (Keith-CY) [inline drizzle/README.md]: `yarn migrate` or `yarn run migrate`?

File: drizzle/doc/zh/README_zh.md [hunk 2]
```diff
@@ -48,7 +54,7 @@ truffle compile
 > 注意: 我们使用 [AppChain-Truffle-Migrate](https://github.com/cryptape/appchain-truffle-migrate) 来部署合约, 所以使用的命令与 truffle-box 有所不同。
 
 ```shell
-npm run migrate
+yarn migrate
```
Comment (Keith-CY) [inline drizzle/doc/zh/README_zh.md]: `yarn migrate` or `yarn run migrate`?

File: drizzle/README.md (hunk 1 above)
Comment (wangwenyue) [reply drizzle/README.md]: yarn migrate for SURE

File: drizzle/doc/zh/README_zh.md (hunk 2 above)
Comment (wangwenyue) [reply drizzle/doc/zh/README_zh.md]: yarn migrate for SURE

**Pull request merged**